rfc/src/rfcs/0012-list.md

```diff
@@ -66,17 +66,37 @@ Totals:
 
 A "Standard Harness" is a `#[proof]` harness, while a "Contract Harness" is a `#[proof_for_contract]` harness.
 
+#### Output Schema
+The formal output schema is:
+```
+Kani Version: float
+Standard Harnesses: List[function] | NONE
+    - function: string
+Contract Harnesses: List[function] | NONE
+    - function: string
+Contracts: Map[function -> contract_harnesses]
+    - function: string
+    - contract_harnesses: List[string] | NONE
+Totals: Map[string -> int]
+    - Standard Harnesses: int
+    - Contract Harnesses: int
+    - Functions with Contracts: int
+    - Contracts: int
+```
+
+All sections will be present in the output, regardless of the result. If a list is empty, Kani will output a `NONE` string.
+
 ### JSON Format
 
-As the name implies, the goal of the `human` output is to be friendly for human readers. If the user wants an output format that's more easily parsed by a script, they can use the `json` option. 
+If the user wants an output format that's more easily parsed by a script, they can use the `json` option. 
 
-The JSON format will contain the same information as the human format, with the addition of file paths and file version. The file version will start at zero and increment whenever we make an update to the format. This way, any users relying on this format for their scripts can realize that changes have occurred and update their logic accordingly.
+The JSON format will contain the same information as the pretty format, with the addition of file paths and file version. The file version will start at zero and increment whenever we make an update to the format. This way, any users relying on this format for their scripts can realize that changes have occurred and update their logic accordingly.
```

Comment:
Should we follow semantic versioning here too? In this case, we probably want to start with `0.1`.